### PR TITLE
Add points field to period

### LINKED ISSUE #95: Add points field to periods
Each period should have a points field, so that each period can have a different amount of points. 
This should remove the current points field from the activity instance, or the current field could be refactored to an initial default points amount.

### PR DESCRIPTION
**Pull request by UndeclaredVariable**
Resolves #95

### CODE REVIEW

**Comment by brendanheywood** [comment]
This did look good but needs a rebase / merge conflict fix

File: classes/persistent/period.php [hunk 1]
```diff
@@ -25,6 +25,7 @@
 namespace mod_cpdlogbook\persistent;
 
 use coding_exception;
+use tool_customlang\local\mlang\langstring;
```
Comment (brendanheywood) [inline classes/persistent/period.php:28]: this doesn't seem right, what this needed for? Generally speaking plugins shouldn't have dependencies on stuff inside other plugins (or for exceptions when they really do that should be declared in the version.php file)

lang_string is declared in moodlelib, vs langstring looks unrelated

Comment (UndeclaredVariable) [reply classes/persistent/period.php]: PHPStorm must've automatically put that `use` there. I'm removed it now.
